tensorwaves/data/generate.py

```diff
@@ -51,9 +51,9 @@ def generate_data(
         [ParticleReactionKinematicsInfo], PhaseSpaceGenerator
     ] = TFPhaseSpaceGenerator,
     random_generator: Callable[
-        [float], UniformRealNumberGenerator
+        [int], UniformRealNumberGenerator
     ] = TFUniformRealNumberGenerator,
-    seed: float = 123456.0,
+    seed: int = 123456,
```

Comment:
Maybe its even best to remove the seed and leave that responsibility to the random_generator instance.

Reply:
Yeah good point

Reply:
Hope 39dcde7 is what you had in mind

Reply:
looks good